python/revng/restapi/__init__.py

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
Soooo, how do we want to call this module? `restapi`? `graphql`?
I'd like to have One True Name. I'd go for either `revng.daemon` or `revng.revngd` and respectively `./bin/revng daemon` or `./bin/revng revngd`.

Reply:
package converted to `revng.daemon`, added command line `revng daemon`